developer/src/kmc-ldml/build.sh

```diff
@@ -74,7 +74,11 @@ fi
 #-------------------------------------------------------------------------------------------------------------------
 
 if builder_start_action test; then
-  npm test
+  eslint .
+  cd test
+  tsc -b
+  cd ..
+  c8 --reporter=lcov --reporter=text mocha "${builder_extra_params[@]}"
```

Comment:
What kind of extra_params could there be?

Reply:
Can pass a specific test to run (i.e. `build.sh test -- my_test_name`)